webapp/components/search_results_item.jsx

```diff
@@ -187,7 +186,7 @@ export default class SearchResultsItem extends React.Component {
         let message;
         let flagContent;
         let rhsControls;
-        if (post.state === Posts.POST_DELETED) {
+        if (post.state === Constants.POST_DELETED) {
```

Comment:
Is this something we need to fix in the Redux repo that `Posts.POST_DELETED` is not the same as `Constants.POST_DELETED`?